src/core/Table/Table.tsx

```diff
@@ -43,17 +43,15 @@ import {
   onExpandHandler,
   onFilterHandler,
   onSelectHandler,
-} from './actionHandlers';
-import { onSingleSelectHandler } from './actionHandlers/selectHandler';
-import {
+  onSingleSelectHandler,
   onTableResizeEnd,
   onTableResizeStart,
-} from './actionHandlers/resizeHandler';
+  TABLE_RESIZE_END_ACTION,
+  TABLE_RESIZE_START_ACTION,
+} from './actionHandlers';
 import VirtualScroll from '../utils/components/VirtualScroll';
 
 const singleRowSelectedAction = 'singleRowSelected';
-const tableResizeStartAction = 'tableResizeStart';
-const tableResizeEndAction = 'tableResizeEnd';
```

Comment:
Don't we want a convention for these?

Reply:
Changed. `tableResizeStartAction` is now exported from Table. These three variable names have also been changed to UPPER_CASE

Reply:
This diff feels unnecessary now.
![image](https://user-images.githubusercontent.com/9084735/162004037-1604dd58-55aa-47dc-b3fb-859c65b93232.png)

Maybe revert the order and convention back to what it was.

Reply:
Reverted order and naming convention